src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -699,4 +699,7 @@ Additional information: {1}</value>
   <data name="Prefer_predefined_type" xml:space="preserve">
     <value>Prefer predefined type</value>
   </data>
+  <data name="Connection_to_remote_host_has_been_lost_some_features_might_stop_working_or_start_working_in_proc_do_you_want_to_recover_remote_host" xml:space="preserve">
+    <value>Connection to remote host has been lost. some features might stop working or start working in proc. do you want to recover remote host?</value>
```

Comment:
right now I made it to crash since not crashing and also not processing diagnostic in proc need more complexity since out of proc is optional feature not mandatory. I need to distinguish whether remote host is not there because service hub terminated or it was never on from the beginning.

by the way, if it was situation where service hub won't be able to be recovered, VS will be crash even with "<re-enable>" button. info bar will only show up when we are in a situation where we can recover service hub.

anyway, made change to crash.